ansible_collections/arista/avd/molecule/eos_cli_config_gen_v4.0/inventory/host_vars/queue_monitor_length.yml

```diff
@@ -0,0 +1,10 @@
+queue_monitor_length:
+  # "enabled: true" will be required in AVD4.0. In AVD3.x default is true as long as queue_monitor_length is defined and not None
```

Comment:
is this accurate? 
```
{% if queue_monitor_length is arista.avd.defined and queue_monitor_length.enabled is not arista.avd.defined(false) %}
```

It looks like for it to work in 3.x it requires:
- `queue_monitor_length` defined
- `queue_monitor_length.enabled` is not `false` (and not `None` as the comment states ?)

Reply:
In 3.x it looks for the `queue_monitor_length` alone, and not any key underneath. That is error-prone in the case of no other keys are required, since the natural yaml `queue_monitor_length:` would assign `None` to the var. So with this we ignore the key unless the sub-key `queue_monitor_length.enabled` is true.

Reply:
maybe I am missing something but the removed code does look for `enabled` to check whether it is `false` or not: `queue_monitor_length.enabled is not arista.avd.defined(false)`.

I am just saying that probably the comment is not accurate. I agree with the change otherwise.

Reply:
Yes, will remove the comment, as claus said below - this is targeted for 4.0 only

Reply:
Updated!